constitution.md

```diff
@@ -0,0 +1,196 @@
+  Involvement
+  - Cosign organization checks with Treasurer
+  - Maintain communication with organization adviser
+
+- Vice-president
+  - Preside over meetings in the absence of the President
+  - Schedule meetings/events with appropriate University offices
+  - Coordinate organization promotion and publicity of events
+
+- Secretary
+  - Maintain an accurate record of all organization meetings and post for
+members
+  - Maintain membership directory
+  - Correspond when necessary with University administration and other
+recognized organizations
+
+- Treasurer
+  - Maintain accurate record of organization transactions
+  - Collect dues if required
+  - Develop organization budget and present to membership for ¾ vote
+  - Cosign organization checks with President
+  - Arrange fundraising opportunities for the organization
+  - Solicits additional funding if needed from the Student Government
+Association in conjunction with the President
+
+Officers may be removed from office by ½ vote of the other officers and ¾ of
+the general membership if actions are deemed inappropriate by the membership.
+The officer is permitted to speak before the Executive Committee and the
+general membership about the charges made concerning his/her performance. The
+officer is not permitted to participate in the deliberation of the Executive
+Committee regarding the charges.
+
+## Article V - Adviser
+
+The adviser shall be a full or part time faculty or staff member at James
+Madison University. The adviser will assume those responsibilities as outlined
+in this constitution and/or found in University Policy No. 3101 entitled
+"Faculty Advisers to Student Organizations." The adviser will be selected by a
+½ vote of the Executive Committee and then presented to the general membership
+for a ¾ vote. Advisers not fulfilling responsibilities or abiding by the
+organization’s purpose may be removed from the position by a ½ vote of the
+Executive Committee. (List basic expectations the organization has of the
+adviser)
+
+## Article VI - Meetings
+
+The organization shall hold regular (weekly) meetings during the academic term
+except when holidays, examination periods or other events make meeting
+impractical.  Organization meeting day/time will be determined in the beginning
+of each semester by a ¾ vote of the general membership. Attendance at
+organization meetings is expected. If a member must miss a meeting,
+correspondence with the secretary is appreciated. A quorum shall consist of a
+simple majority of the membership plus two officers. Robert’s Rules of Order
+(newly revised) shall govern all meetings.
+
+## Article VII - Finance
+
+The organization may establish reasonable dues that must be paid by all members.
+The amount of the dues will be determined in the beginning of the academic year
+by the Executive Committee and presented to the general membership for a ¾ vote.
+Dues must be paid by (enter week in semester). The treasurer shall maintain all
```

Comment:
An actual week would need to be specified here.

Reply:
Maybe require at least one business meeting in the first two weeks of each semester for officers/members from the previous semester to set a draft agenda and due amounts for the upcoming semester, and then have dues paid by week 4 or 5?